### ISSUE TITLE
Update samples

### ISSUE THREAD

**Issue by rtabbara** [mono/CocosSharp#392]
As mentioned [here](https://github.com/mono/cocos-sharp-samples/issues/16), we need to bring the samples up to CocosSharp v1.7+ &mdash; primarily handling the change from `CCApplication` to `CCGameView` .

Additionally, dependent on #389, it would be ideal of the samples incorporated the MG content pipeline rather than pre-built platform-specific assets.

**Comment by eatdrinksleepcode** [comment]
Yes please. Accidentally seeing this issue is the first time I realized that when I started my new game based on existing tutorials using CCApplication that I was actually starting on a deprecated foundation (worse than deprecated, apparently already removed). _Everything_ I can find online uses the "old way"; if I wanted to use the new way, I wouldn't even know where to start looking for guidance on how to do it.
